<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleWindow</class>
 <widget class="QMainWindow" name="ModuleWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Finance module</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Set date range</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="sort_fromdateLabel">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dateEditFrom">
         <property name="date">
          <date>
           <year>2023</year>
           <month>3</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="sort_todateLabel">
         <property name="text">
          <string>Till</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="dateEditTo">
         <property name="date">
          <date>
           <year>2023</year>
           <month>4</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="sort_button">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="income_range_label">
         <property name="text">
          <string>Income in range:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="income_range_lineedit">
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="expense_range_label">
         <property name="text">
          <string>Expense in range:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="expense_range_lineedit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="total_label">
         <property name="text">
          <string>Total</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="total_lineedit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="income_deletebutton">
         <property name="text">
          <string>Delete income row</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="expense_deletebutton">
         <property name="text">
          <string>Delete expense row</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QPushButton" name="add_data_button">
         <property name="text">
          <string>Add Data</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QPushButton" name="revert_button">
         <property name="text">
          <string>Revert</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Expenses</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="expense_tableView">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Income</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="income_tableView">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionEditable_tables_data"/>
   </widget>
   <addaction name="menuedit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMenu">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="actionEditable_tables_data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Editable tables data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
